pipelines.md

```diff
@@ -123,7 +123,7 @@ results/figures:
 ```
 
 ```{margin}
-`make` can be pretty finicky. For instance, it requires using tabs to indent and does not support spaces.
```

Comment:
I found this sentence a bit confusing - it does support spaces, just not for indentation. I tried rewriting it to be a bit clearer.